src/task_1.1/task1.js

```diff
@@ -0,0 +1,22 @@
+const process = require('process');
+const stdin = process.stdin;
```

Comment:
Such variables could be considered redundant since they don't bring any additional meaning.
Moreover, it's best to avoid using reserved words as names for variables.